src/test/java/selenium/LoginTest.java

```diff
@@ -0,0 +1,47 @@
+package selenium;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static selenium.Locators.*;
+
+public class LoginTest {
+
+    private WebDriver driver;
+    private TestUser testUser;
+
+    @BeforeEach
+    public void setup() {
+        driver = new ChromeDriver();
+        testUser = new TestUser(System.getenv("TEST_USER"), System.getenv("TEST_PASSWORD"));
+    }
+
+    @Test
+    @DisplayName("Login test for saucedemo site")
+    public void loginTest() {
+        driver.get("https://www.saucedemo.com/");
+        WebElement usernameInput = driver.findElement(USER_NAME_INPUT_LOCATOR);
+        System.out.println(testUser.getUserName());
```

Comment:
Let's use some logger (logback or log4j2) instead of printing to console using this method

Reply:
Done